ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml

```diff
@@ -102,3 +102,37 @@
     - nvidia_cuda_toolkit
     # TODO: Package installs should not use latest
     - skip_ansible_lint
+
+- name: Download NVidia CUDA Repo RPM - Ubuntu 16 on PPC64LE
```

Comment:
Does this need to be locked to Ubuntu 16? i.e. is there a reason not to install on any Ubuntu 18.04 systems?

Reply:
The nvidia repo url is o/s version and platform specific, same as the key; `ubuntu1604/ppc64el`

Reply:
Disappointing :'(